lightspeed/assemblies/assembly_configuring-lightspeed-onpremise.adoc

```diff
@@ -0,0 +1,36 @@
+ifdef::context[:parent-context: {context}]
+
+:_content-type: ASSEMBLY
+
+[id="configuring-lightspeed-onpremise_{context}"]
+
+= Setting up {LightspeedShortName} on-premise deployment
+
+:context: configuring-lightspeed-onpremise
+
+[role="_abstract"]
+
+As an {PlatformName} administrator, you can set up {LightspeedShortName} on-premise deployment and connect it to an {ibmwatsonxcodeassistant} instance. After the on-premise deployment is successful, you can start using the Ansible Lightspeed service by using Ansible Visual Studio (VS) Code extension. 
```

Comment:
Missing article in the first sentence: 
...you can set up **a** Red Hat Ansible Lightspeed on-premise deployment...

Suggested rewrite for the second sentence to avoid using the word "using" twice: 
After the on-premise deployment is successful, you can start using the Ansible Lightspeed service with the Ansible Visual Studio (VS) Code extension.